types/dagre/index.d.ts

```diff
@@ -12,15 +12,27 @@ export namespace graphlib {
         nodes(): string[];
         node(id: any): any;
         setDefaultEdgeLabel(callback: () => void): Graph;
-        setEdge(sourceId: string, targetId: string, options?: { [key: string]: any }): Graph;
+        setDefaultNodeLabel(callback: () => void): Graph;
+        setEdge(sourceId: string, targetId: string, options?: { [key: string]: any }, value?: string): Graph;
```

Comment:
It seems like `setEdge` has another [overload](https://github.com/cpettitt/dagre/blob/master/dist/dagre.js#L3795).